### ISSUE TITLE
General organization

### ISSUE THREAD

**Issue by jpellman** [dylanrees/citizen-science#8]
I just noticed this repository and think that its existence is long overdue.  I actually started a citizen science list of my own a couple days ago based off the (currently uneditable) wiki at openhatch.org.  One thing that I noticed about the approach that you've taken with this list vs the approach that I've started to take is that this list is much more focused on tools, whereas my idea was to focus on scientific disciplines themselves and then further divide them into 3 categories: data collection, grid computing, and human computation.  My approach, however, might be seriously lacking in the philosophy department.  Also, did we go to college together?

**Comment by dylanrees** [comment]
Wow, we did go to college together!  I don't remember meeting you but that's pretty cool.

For now I think the list format is fine, but like you, I've been thinking of ways this information might be organized better as it grows more extensive.  I agree that it would be nice to be able to list things by discipline, although I also don't want to lose the ability sort the information by tool/resource type.  Here's a rough sketch of one possibility I was considering:
- The list stays the way it is, but with tags added to the end of each entry to categorize by discipline.  #physics, #biology, and so forth.  This way there continues to be a nice GitHub-based list that is human-readable and easily editable.
- However, a platform gets built to parse this information into .json or some other nice data format and serve with a nice interactive frontend that allows you to choose all resources corresponding to whatever discipline(s) and tool type(s) you want. For instance, if I want to find all equipment and all spaces that are for synthetic biology, ecology and geology, I click the appropriate buttons and the frontend spits out all the info that matches that.

**Comment by jpellman** [comment]
I don't know if I agree with this approach.  The whole appeal of the awesome lists I've come across is their simplicity- an easily editable document written in a markup language and nothing else.  The possibility you've proposed increases the scope of your awesome list dramatically from just a markup language document to an entire software stack where a text document is parsed into a JSON, read back in by a web framework, and then served up by an http service.  It also creates heavy overlap with websites that already exist, such as [SciStarter](scistarter.com) and would result in a lot of duplicated effort.  Ideally, the value of a citizen science awesome list over what already exists would be the ease with which someone could contribute, which would effectively make such a list slightly ahead of the curve in terms of curation.

In general, I think it might be a good idea to revisit the scope of this list.  Some items are very clearly relevant to citizen science, while others are more tangential (e.g., LibreOffice, which while useful, is probably the last thing I think of when I go into the lab to do work; I do use it, it's just not a core part of what I do).  It also might be useful to make a distinction between more active and passive levels of engagement.  The list as it stands now seems to be more targeted towards people who want to build and fund their own lab independently (similar to the gentleman scientist paradigm of the 18th century), while a lot of the projects I've been interested in require much less of a time commitment since they are managed by institutions such as Cornell or UC Berkeley.

**Comment by dylanrees** [comment]
Yeah, as I've continued to think about ways to improve this list, I've been moving away from the software stack idea I described in my last reply.  I don't think I want to list the resources by scientific discipline if it comes at the expense of resource type (journal, software, equiment, etc.), partially because of the sheer number of disciplines we could list and partially because of the way some resources fall into multiple disciplines, or no particular discipline.

I do agree that LibreOffice and a few of the other tools might not be particular enough to citizen science.  However, I think having access to open-source office and productivity tools is generally a good thing for citizen scientists.  If there's another list or repository somewhere that caters specifically to these software tools and makes it easy to find the one you want, I might link it and delete some of the links I currently have.

I can tell that your interests within the space of citizen science don't align 100% with mine, but please add any interesting links you find!  If need be, we can discuss categorization further as the list evolves.

Dylan
